handbook/product.md

```diff
@@ -334,6 +334,9 @@ Use `---`, with color `$ui-fleet-black-50` as the default UI for empty columns.
 
 Pressing the return or enter key with an open form will cause the form to be submitted.
 
+**External links**
+
+When adding an external link to text, use `$core-blue` as the color and `xs-bold` styling. The link-out icon will also be included to the right of the text.
```

Comment:
```suggestion
For a link that navigates the user to an external site (ex. fleetdm.com/docs), use the `$core-blue` color and `xs-bold` styling for the link's text. Also, place the link-out icon to the right of the link's text.
```

I propose tweaking the language here so that it first communicates what these links are used for (navigating to an external site). What do you think?

Reply:
Yes! I like these changes.